src/mcts/search.cc

```diff
@@ -207,8 +207,7 @@ std::vector<std::string> Search::GetVerboseStats(Node* node,
                                                  bool is_black_to_move) const {
   const float fpu = GetFpu(params_, node, node == root_node_);
   const float cpuct = ComputeCpuct(params_, node->GetN());
-  const float U_coeff =
-      cpuct * std::sqrt(std::max(node->GetChildrenVisits(), 1u));
```

Comment:
I think now we can delete GetChildrenVisits() completely.

Reply:
Its still used in GetV4TrainingData.

Reply:
It can be `GetN() - 1` there probably. :)